scripts/db_migrator.py

```diff
@@ -756,10 +800,19 @@ def version_2_0_1(self):
 
     def version_2_0_2(self):
         """
-        Version 2_0_2
-        This is the latest version for 202012 branch 
+        Version 2_0_2.
         """
         log.log_info('Handling version_2_0_2')
+        # Update cable length data and re-assign interface speed for T0 devices with interfaces connected to EdgeZone Aggregators
+        self.update_edgezone_aggregator_config()
+        self.set_version('version_2_0_3')
```

Comment:
Why are we updating here and many other places? This is only needed on 5_0_0. Let's discuss offline